server_installation/topics/operator/keycloak-client-cr.adoc

```diff
@@ -6,7 +6,7 @@ You can use the Operator to create clients in {project_name} as defined by a cus
 
 [NOTE]
 ====
-You can update the YAML file and changes appear in the {project_name} admin console, however changes to the admin console do not update the custom resource.
+You can create and delete the YAML file and changes appear in the {project_name} admin console, however changes to the admin console are not reflected back and updates of the CR are not supported.
```

Comment:
This is not accurate. Client and User support limited updatability.

Reply:
removed

Reply:
@vmuzikar before I pick up this PR again, do you think that we can improve the wording according to the comment above?

Reply:
Ah, sorry, missed this comment. Clients really support update, so we should not just drop that now.

Maybe we could just update the wording there, using "limited updates support" or something like that.